### DISCUSSION TITLE
Ideas for disseminating USER project findings

### DISCUSSION THREAD

**Discussion by meagdoh** [sprblm/USER_project#27] [Ideas]
- A special issue of the[ Journal of Open Research Software](https://openresearchsoftware.metajnl.com/) focused on the state of usability and human-centered design in research software

- Readouts to designers 
  - NIH All of Us UX team
  - Host a webinar, and invite [Design Gigs for Good](https://www.designgigsforgood.org/), [International Design in Government](https://international.gov-design.com/), [Turing Way](https://the-turing-way.netlify.app/community-handbook/fireside-chat.html), 

- Readouts to RS teams/executives 
  - Host a webinar, and invite [Microsoft Research UK](https://www.microsoft.com/en-us/research/lab/microsoft-research-cambridge/), [people who attended the recent RSE in Data & AI Workshop](https://warwick.ac.uk/research/data-science/warwick-data/news/events/research-software-engineering-workshop/),

**Comment by meagdoh** [comment]
@smklibrarian, here are a few of the ideas for dissemination. I am also tracking additional discussions about upcoming talks here: https://github.com/meagdoh/ssi-fellowship/issues/10

**Comment by meagdoh** [comment]
- Focus group of experts to do first reads and provide feedback

**Comment by meagdoh** [comment]
Turing Way for Open UX design

**Comment by meagdoh** [comment]
OR a chapter on Turing Way

**Comment by smklibrarian** [comment]
@Erioldoesdesign sharing this thread with you as we loop you in to the dissemination ideas for upcoming conferences and opportunities to share the findings!

**Comment by Erioldoesdesign** [comment]
Loving this thread! Would love to talk about this post-writing and content creation. I feel like we're very much in the collating phase of all of the resources, thoughts, findings etc. from our research so let's get to a place where we feel like we're ready to pause that part and then assess these options?

I'll add some options in separate comments!

**Comment by Erioldoesdesign** [comment]
Extending the Zine making workshops to be 'read a chapter and then make a Zine summary of the chapter'

Goal: Helps us to understand what parts of a chapter landed for individuals and what the highlight takeaways are and also what was avoided/too difficult

**Comment by Erioldoesdesign** [comment]
1. A way of assessing maturity of design and usability in S&R OSS projects - questionnaire, assessment quiz, intake conversation, maturity/readiness ladder or a combo or all of the above etc.
2. Carpentries workshop/a workshop framework for different stages of 'maturity' of S&R OSS tools

Goal: In speaking with various tools and teams we found that there are wildly different circumstances where different kinds of design efforts would best improve usability and design with complicated competing factors impacting maturity/readiness levels (no surprises here! /jk)
If we can begin to compile a way of helping project assess where they are on a practical, team organisational and conceptual level re. usability and design we can then begin to understand where to 'place' them in a series of resources and/or activities, guides and workshops that best address their needs

Honestly it's sounding a bit like what we tried to do with both the [Design Inventory (with UNICEF)](https://unicef.github.io/inventory/design/) and also a [The Dev's guide to...](https://simplysecure.github.io/devs-guide-to/)

**Comment by Erioldoesdesign** [comment]
Adding the ideas post Jupyter con conversations with Meag and other attendees of JupyterCon:

- Tools and training for conducting mixed methods research within academia plus pathways to merge research activities. Example: I am sitting with OSS product manager who wants to learn more about Jr researchers. So do I at All of Us…. What are the pathways we can enable joint user research efforts?

- Design as a sustainable feature of SROSS. You can’t just pay money and set free. Need better understanding of how to do this sustainably where SROSS projects are welcoming/have dev support to act on ideas.

- Building the case for the professionalization of design in research and OSS. See RSE movement. Where’s the RSDesigner?

- Funds for a convening of ux people in science (there’s an enthusiastic critical mass) and / or presenting the work at a convening that’s self organized ? Also more practical next steps: shadowing teams, testing recommendations with specific teams, developing curricula etc

**Comment by Erioldoesdesign** [comment]
interested in funds for:
	•	Practical next steps:
	◦	shadowing teams
	◦	testing recommendations with specific teams 
	◦	developing curricula etc
	•	How to partner with organisations and community groups like open source design .net to bring in more science and research understanding into the design OSS space which can also then facilitate better outreachy and google summer of code internships that are focussed on user research and design projects -> this would hopefully have a knock on effect of adding to or multiplying the amount of skilled designers in science and research which is an underserved knowledge space.
